### ISSUE TITLE
ControlStructureSpacing gets confused with multiline comments

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#520]
While trying the `ControlStructureSpacing` sniff with the following configuration:
```xml
    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
      	<properties>
        		<property name="tokensToCheck" type="array">
          			<element value="T_RETURN"/>
        		</property>
      	</properties>
    </rule>
```

And the following code:
```php
    protected function foo(): string
    {
        // Lorem Ipsum
        // Ipsum
        // Lorem
        return 'Foo bar baz';
    }
```

I got the following error:
```
FILE: Kameleon/Product/Price/Calculator.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 233 | ERROR | [x] Expected 1 lines before "return", found 3.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

Witch IMHO isn't right, as there's no new line between the comments and the return statment.
